src/overlays/actors/ovl_En_Fall/z_en_fall.c

```diff
@@ -693,7 +696,7 @@ void EnFall_Moon_Draw(Actor* thisx, GlobalContext* globalCtx) {
     Matrix_MultiplyVector3fByState(sFocusOffset, &this->actor.focus.pos);
     primColor = (this->eyeGlowIntensity * 200.0f) + 40.0f;
```

Comment:
I think a space between the matrix and the primcolor set would make this a bit clearer. Applies elsewhere.